frontend/package.json

```diff
@@ -47,7 +50,7 @@
     "web-vitals": "^0.2.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "set PORT=3000 && react-scripts start",
```

Comment:
Why are we setting port from package.json ?
Where was it earlier and why do we need this change ? Just curious.